src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/group/RabbitMessageGroupBatchRouter.kt

```diff
@@ -39,77 +43,76 @@ class RabbitMessageGroupBatchRouter : AbstractRabbitRouter<MessageGroupBatch>()
 
     override fun splitAndFilter(
         message: MessageGroupBatch,
-        pinConfiguration: @NotNull QueueConfiguration
+        pinConfiguration: @NotNull QueueConfiguration,
+        pinName: PinName
     ): @NotNull Collection<MessageGroupBatch> {
         val builders = mutableListOf<Builder>()
```

Comment:
Please check that filters aren't empty before execute other logic

Reply:
done